MapboxDirections/MBRouteStep.swift

```diff
@@ -623,6 +626,8 @@ open class RouteStep: NSObject, NSSecureCoding {
         destinations = decoder.decodeObject(of: [NSArray.self, NSString.self], forKey: "destinations") as? [String]
         
         intersections = decoder.decodeObject(of: [NSArray.self, Intersection.self], forKey: "intersections") as? [Intersection]
+        
+        voiceInstructions = decoder.decodeObject(of: [NSArray.self, Intersection.self], forKey: "voiceInstructions") as? [VoiceInstruction]
```

Comment:
This will fail because the array contains `VoiceInstruction`s, not `Intersection`s.